.codecov.yml

```diff
@@ -0,0 +1,111 @@
+# codecov:
```

Comment:
we can lose all the comments in this file, if you really want you can leave one comment with a link to whatever example you pulled this out of

Reply:
removed them